MANIFEST.in

```diff
@@ -13,6 +13,8 @@ include *requirements.txt
 include tox.ini
 include run-tests.sh
 
+recursive-include Lib/fontTools py.typed
```

Comment:
I think `recursive-include` is for globs or multiple files that can appear in sub-paths, if it's a single file you can just do the normal `include`

Reply:
Yas but what if we start to include these files all over?

Reply:
I thought you only needed one at the root. But you know better

Reply:
if everything below is typed, yes. https://www.python.org/dev/peps/pep-0561/#packaging-type-information

> Package maintainers who wish to support type checking of their code MUST add a marker file named py.typed to their package supporting typing. This marker applies recursively: if a top-level package includes it, all its sub-packages MUST support type checking as well.

Reply:
gotcha, ty